collect_app/src/main/java/org/odk/collect/android/gdrive/InstanceGoogleSheetsUploader.java

```diff
@@ -14,7 +14,7 @@
 
 package org.odk.collect.android.gdrive;
 
-import static org.odk.collect.android.javarosawrapper.FormController.INSTANCE_ID;
+import static org.odk.collect.android.javarosawrapper.JavaRosaFormController.INSTANCE_ID;
```

Comment:
I think it'd be good to get not need to depend on `JavaRosaFormController` to grab these constants, but we can look at rearranging that another time. Adding the interface is a great change for the moment!